packages/invoices/src/OutboundShipment/DetailView/modals/ItemDetailsForm.tsx

```diff
@@ -1,75 +1,58 @@
 import React from 'react';
 import {
-  FieldValues,
   Grid,
   Item,
-  ModalInput,
+  BasicTextInput,
```

Comment:
Similarly here, instead of a summary item, just an `Item`, removed the `register`  but that's about it